go.mod

```diff
@@ -219,7 +219,7 @@ replace (
 	github.com/coreos/bbolt => github.com/coreos/bbolt v1.3.1-coreos.6
 	github.com/coreos/etcd => github.com/coreos/etcd v3.3.13+incompatible
 	github.com/coreos/go-etcd => github.com/coreos/go-etcd v2.0.0+incompatible
-	github.com/coreos/go-oidc => github.com/coreos/go-oidc v0.0.0-20180117170138-065b426bd416
+	github.com/coreos/go-oidc => github.com/coreos/go-oidc v2.0.1-0.20190815175729-2be1c5b8a260+incompatible
```

Comment:
I won't stick on it since we weren't on a tag before, but any chance of getting a v2.1.0 tag cut for this? cc @ericchiang

Reply:
Yeah. I'll do that today

Reply:
Tagged https://github.com/coreos/go-oidc/releases/tag/v2.1.0 :)